pkg/didcomm/dispatcher/api.go

```diff
@@ -35,3 +53,11 @@ type Outbound interface {
 	// Forward forwards the message without packing to the destination.
 	Forward(interface{}, *service.Destination) error
 }
+
+// NoOPMessageServiceProvider returns noop message service provider
+type NoOPMessageServiceProvider struct{}
```

Comment:
This should be an unexpected member of the aries framework package.
It is not an API.

Reply:
fixed